Makefile

```diff
@@ -481,7 +481,7 @@ build-ci:
 # - node-test-commit-linux-coverage: where the build and the tests need
 #   to be instrumented, see `coverage`.
 run-ci: build-ci
-	$(MAKE) test-ci
+	$(MAKE) test-ci -j1
```

Comment:
Is this WIP?

Reply:
Yes, and no. for the canary branch I float this as https://github.com/nodejs/node/pull/23286/commits/6e24dcca961b36d22351a62757717bd70783de30 
It's either this or https://github.com/nodejs/node/pull/23257. One of them will need to land upstream, since now we see https://github.com/nodejs/node/issues/22006 on a lot of platforms (`make test-ci` rebuilds a few files and relinks the node binary, in parallel to the script that builds the addons, so it's a race to the binary)

Reply:
> It's either this or #23257.

I’m not sure I follow … that PR landed, right?

I don’t think `-j1` is something we want in CI…

Reply:
> I’m not sure I follow … that PR landed, right?

#23257 was a revert of https://github.com/nodejs/node/commit/5d8373a498a50b1387464391402ef22636439303, because there was a bug on macOS.

IMHO running just the `make` part of `test-ci` with `-j1` is not that bad. Most of the steps have parallelism built in https://github.com/nodejs/node/blob/2ba19ff2b8a6253c6fb7795f35d4fb1c30ec6c30/Makefile#L457-L467

For `build-addons` && `build-addons-napi` you added the script that does multi-proc. `doc-only` had it for a long time. and the test uses `$JOBS` for parallelism level...